config/webpack.dev.js

```diff
@@ -3,6 +3,8 @@ const common = require("./webpack.common.js");
 const HtmlWebpackPlugin = require("html-webpack-plugin");
 const fs = require("fs");
 
+const htmlLayout = fs.readFileSync("./site/layout.html");
```

Comment:
_loader-update_: Set the `"utf-8"` flag so the file is read as a `string` (instead of `Buffer`)

```suggestion
const htmlLayout = fs.readFileSync("./site/layout.html", "utf-8");
```